lib/travis/api/serialize/v2/http/user.rb

```diff
@@ -37,6 +37,7 @@ def user_data
                   'synced_at' => format_date(user.synced_at),
                   'correct_scopes' => Github::Oauth.correct_scopes?(user),
                   'created_at' => format_date(user.created_at),
+                  'last_logged_in_at' => format_date(user.last_logged_in_at),
```

Comment:
is this meant to be `first_logged_in_at`

Reply:
hahaaaaaaaaaaaaaaaaa

sooooo tiiiiiiiiiiiiiiiiired